packages/core/saved-objects/core-saved-objects-common/src/saved_objects_imports.ts

```diff
@@ -91,6 +91,7 @@ export interface SavedObjectsImportFailure {
    * If `overwrite` is specified, an attempt was made to overwrite an existing object.
    */
   overwrite?: boolean;
+  managed?: boolean;
```

Comment:
Same question here, I don't see it as being a problem, but just so I understand, why do we need to surface the `managed` flag on SO import failures/successes?

Reply:
As a reference, that's all.